docs-source/content/userguide/managing-domains/domain-lifecycle/startup.md

```diff
@@ -153,6 +153,17 @@ To shut down a specific standalone server, add it to the Domain and set its `ser
       serverStartPolicy: "NEVER"
     ...
 ```
+{{% notice note %}}
+The Administration Server can be shut down by setting the `serverStartPolicy` of the `adminServer` to `NEVER`.
+Care should be taken when shutting down the Administration Server, as it may not be able to start up 
```

Comment:
I suggest changing the wording of the note to this, "Care should be taken when shutting down the Administration Server. When not available, the Administration Server cannot start up Managed Servers in the WebLogic domain. If it cannot connect to the Administration Server during startup, a Managed Server will try to start up in Managed Server Independence (MSI) mode but this could fail due to reasons such as no accessible Authentication Provider from the Managed Server pod."

Reply:
Also, do keep the cross-references to the WLS docs.

Reply:
Thanks for the suggestion! 

However, this sentence "When not available, the Administration Server cannot start up Managed Servers in the WebLogic domain." is not entirely correct. It gives the impression that the managed servers are started by the admin server which is not right. It is also possible for the managed servers to start up successfully in some cases, such as if the domain is configured to use some external authentication provider that can be accessed by the managed server pods.

What do you think if we remove that sentence and update to: "Care should be taken when shutting down the Administration Server. If a Managed Server cannot connect to the Administration Server during startup, it will try to start up in Managed Server Independence (MSI) mode but this could fail due to reasons such as no accessible Authentication Provider from the Managed Server pod."

Thanks!

Reply:
Yes, perfect.